packages/insomnia-cookies/src/tough-cookie.d.ts

```diff
@@ -0,0 +1,244 @@
+// These types are adapted from the v2.3.5 types because module augmentation is not possible because they named a namespace (`CookieJar`) the same name as a namespace (`CookieJar`) which causes a ciruclar dependency when you try to augment the CookieJar class to add some of the (missing) types we need.
```

Comment:
```suggestion
// These types are adapted from the v2.3.5 types because module augmentation is not possible because they named a namespace (`CookieJar`) the same name as a namespace (`CookieJar`) which causes a circular dependency when you try to augment the CookieJar class to add some of the (missing) types we need.
```